Documentation/guides/BuildProcess.md

```diff
@@ -1135,6 +1135,29 @@ server, the following MSBuild properties can be used:
 
     Note: `env:` is not supported when using `AndroidPackageFormat`=`aab`. 
 
+-   **JarsignerTimestampAuthorityUrl** &ndash; This property
+    allows you to specify a Url to a timestamp authority
+    service. This can be used to make sure your apk signature
```

Comment:
Tiny formatting idea: It looks like when referring to a file, `apk` mostly appears in this document as `` `.apk` ``, so it could be written as `` `.apk` `` here too.